### ISSUE TITLE
Any way to clear the MM panel? 

### ISSUE THREAD

**Issue by viraj89** [joeferraro/MavensMate#640]
Is there any way to clear the MM panel in sublime?

The panel eventually fills with several error, output etc messages and it goes on. Example I have 600+ lines and is there any way to clear it? 

![12](https://cloud.githubusercontent.com/assets/9387490/13070711/7ccd5efa-d4b4-11e5-937f-ef3c913727d7.PNG)

**Comment by pbattisson** [comment]
@viraj89 You can hide the panel by hitting escape on the keyboard

**Comment by viraj89** [comment]
@pbattisson Yes, with Esc or Cntl+Shift+M I can toggle it, but any way to clear the whole content with restarting the project? 

As we use `clear` in command prompt to clear the screen, anything something that I'm unaware off to clear the stuff from the panel?
